### ISSUE TITLE
The Networking TS

### ISSUE THREAD

**Issue by Type1J** [eidheim/Simple-Web-Server#223]
The library can use standalone ASIO and Boost ASIO. Because ASIO has a new incarnation in the C++ standard itself (The Networking TS) AND [a compatibility layer](https://www.boost.org/doc/libs/1_66_0/doc/html/boost_asio/net_ts.html), the Networking TS should be the sole target in future versions. This should make Simple-Web-Server more portable and I believe removes ALL external dependencies.

**Comment by eidheim** [comment]
Yes this is the end goal, but we have to also consider older compilers and 5-year LTS's. However, in the meanwhile, it might be that we can support `std::net` as we do both `asio` and `boost::asio` Today.
